src/CSModule.sol

```diff
@@ -65,7 +68,8 @@ contract CSModule is
 
     ICSAccounting public accounting;
 
-    ICSEarlyAdoption public earlyAdoption;
+    /// @dev deprecated
+    address internal _earlyAdoption;
```

Comment:
shall we add a test for checking storage? just in case

Reply:
what this test should test?

Reply:
That the storage won't corrupted after update

Reply:
this would be a part of a separate upgrade tests I believe